### ISSUE TITLE
[Bug]: unable to launch netclient gui because no opengl windows server

### ISSUE THREAD

**Issue by si458** [gravitl/netmaker#1145]
### What happened?

```
PS C:\Users\Administrator\Downloads> .\netclient.exe
2022/05/28 17:31:13 Gravitl Netclient on Windows started
2022/05/28 17:31:13 Fyne error:  window creation error
2022/05/28 17:31:13   Cause: APIUnavailable: WGL: The driver does not appear to support OpenGL
2022/05/28 17:31:13   At: C:/Users/alexf/go/pkg/mod/fyne.io/fyne/v2@v2.1.4/internal/driver/glfw/driver.go:123
```

### Version

v0.14.1

### What OS are you using?

Windows

### Contributing guidelines

- [X] Yes, I did.

**Comment by afeiszli** [comment]
What windows version, and is it a VM?

**Comment by si458** [comment]
> What windows version, and is it a VM?

Server 2012 r2 and no it's a physical machine

**Comment by si458** [comment]
i also get this error on a physical Server 2016 machine too,
only thing i can think of is the both have no graphics drivers installed (its currently showing as Microsoft Basic Display Adapter)
both servers are quite old and the is no graphics drivers for 2012 r2 upwards for them both, but i normally use RDP anyways

**Comment by afeiszli** [comment]
Yeah it’s definitely the drivers. We may need to build a GUI-less version for windows.

**Comment by afeiszli** [comment]
Try running it from the CLI though. I believe it should still work. You just can’t launch the gui.

**Comment by si458** [comment]
> Try running it from the CLI though. I believe it should still work. You just can’t launch the gui.

the cli works no problem even joining, using, etc, its just running it normally for the GUI that just closed straight away,
until you run it on command prompt to realise it wont load the GUI

could you maybe put a CATCH method in that displays a message when you run the GUI saying 'computer not supporting GUI' ?
rather then running the desktop icon and nothing happening?

**Comment by afeiszli** [comment]
ok, yes I believe that's an appropriate fix for this

**Comment by aniston** [comment]
@si458 , you mentioned that you are running on Physical machines, so probably the following will not apply to you, but for a VM on a VirtualBox host on windows..

- setting the Display driver as VBoxSVGA with 3D acceleration and 128MB vRAM (+Guest Additions in the VM) got me the GUI loaded.
![image](https://user-images.githubusercontent.com/3067891/214703163-33b43121-b23e-434e-9ac0-06aac4561a61.png)
